roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -58,7 +59,7 @@
   timeout: 30
   retries: 20
   until: '"CREATE_COMPLETE" in result.stdout'
-    #  failed_when: '"CREATE_COMPLETE" not in result.stdout'
+  failed_when: '"CREATE_COMPLETE" not in result.stdout'
```

Comment:
I think this is not strictly necessary, since the task fails when the number of retries reaches the limit